app/scripts/modules/core/src/managed/ManagedResourceObject.tsx

```diff
@@ -115,18 +97,21 @@ export const ManagedResourceObject = memo(
       ) : undefined;
 
     return (
-      <ObjectRow
-        icon={resourceManager.getIcon(kind)}
-        title={linkProps ? <a {...linkProps}>{displayName}</a> : displayName}
-        depth={depth}
-        content={resourceStatus}
-        metadata={
-          <span className="flex-container-h middle">
-            {currentPill}
-            {deployingPill}
-          </span>
-        }
-      />
+      <div className="ObjectRow" style={{ marginLeft: 16 * depth }}>
```

Comment:
What is the reason for deleting the `ObjectRow` component? The code here seems the same, but I also don't have much context.

Reply:
It was in use only here, and it added another layer of abstraction for no clear reason (in addition `ObjectRow` is too vague imo)